templates/design.md

```diff
@@ -0,0 +1,94 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty, to be filled in later)
+- Authors: (the names of everyone contributing to this RFC)
+
+# (RFC title goes here)
+
+## Summary
+
+<!-- One paragraph explanation of the feature. -->
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   to be able to implement it by reading this document. Please get into specifics
```

Comment:
This should probably start with: "can implement it"

Full (proposed) sentence: "Explain the design with enough detail that someone familiar with ESLint can implement it by reading this document."